csharp/Server/App.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using Cassandra; // DataStax Cassandra C# driver
+
+namespace CarePet.Server
+{
+    public class App
+    {
+        public static void Main(string[] args)
+        {
+            var config = Config.Parse(new Config(), args);
+
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Register Config as singleton
+            builder.Services.AddSingleton(config);
+
+            // Register Cassandra session as singleton
+            builder.Services.AddSingleton<ISession>(sp =>
+            {
+                var cfg = sp.GetRequiredService<Config>();
+                return cfg.Builder(Config.Keyspace).Build().Connect(Config.Keyspace);
+            });
+
+            // Add controllers (API endpoints)
```

Comment:
I don't see any point in having these comments they basically repeat what code says